### DISCUSSION TITLE
Any Idea how to achieve this rather advanced attack ? (similar to traffic analysis attack)

### DISCUSSION THREAD

**Discussion by minanagehsalalma** [abhinavsingh/proxy.py#1319] [Ideas]
I made a video demonstrating the idea

https://user-images.githubusercontent.com/20546638/224743214-a866e0f4-4afc-432c-9608-e00f6274c4b4.mp4

what i am lost at is ... how to know if a user just opened an app like FB for example... so the attack becomes convenient

**Comment by abhinavsingh** [comment]
With TLS Interception enabled https://github.com/abhinavsingh/proxy.py#tls-interception you should be able to inspect all the request attributes.   You can use one of the existing plugin as basis of your own custom plugin.

**Reply by minanagehsalalma** [reply]
> You can use one of the existing plugin as basis of your own custom plugin.

@abhinavsingh what plugins do you suggest ?
